src/wp-includes/class-wp-recovery-mode.php

```diff
@@ -262,6 +263,21 @@ public function clean_expired_keys() {
 		$this->key_service->clean_expired_keys( $this->get_link_ttl() );
 	}
 
+	/**
+	 * Unschedule the recovery_mode_clean_expired_keys Event.
+	 */
+	public function remove_orphaned_recovery_mode_cron_event() {
+		// Check if we are in a Multisite installation.
+		if ( is_multisite() ) {
+			// Check if the cron event is scheduled.
+			$timestamp = wp_next_scheduled( 'recovery_mode_clean_expired_keys' );
```

Comment:
Hello @narenin,

I think it is better to use the `wp_clear_scheduled_hook` function instead of the `wp_unschedule_event`.

@mukeshpanchal27 Please share your opinions as well.

Thanks